src/Model/CustomerTokenService.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/module-customer-graph-ql
+ * @link https://github.com/scandipwa/module-customer-graph-ql
+ */
+
+declare(strict_types=1);
+
+namespace ScandiPWA\CustomerGraphQl\Model;
+
+use Magento\Customer\Api\AccountManagementInterface;
+use Magento\Framework\Exception\EmailNotConfirmedException;
+use Magento\Integration\Model\CredentialsValidator;
+use Magento\Integration\Model\Oauth\TokenFactory as TokenModelFactory;
+use Magento\Integration\Model\ResourceModel\Oauth\Token\CollectionFactory as TokenCollectionFactory;
+use Magento\Integration\Model\Oauth\Token\RequestThrottler;
+use Magento\Framework\Exception\AuthenticationException;
+use Magento\Framework\Event\ManagerInterface;
+use Magento\Integration\Model\CustomerTokenService as SourceCustomerTokenService;
+
+/**
+ * Class CustomerTokenService
+ * @package ScandiPWA\CustomerGraphQl\Model
+ */
+class CustomerTokenService extends SourceCustomerTokenService
```

Comment:
well, this is generally fine, but we're also rewriting 100% of the original class methods and properties

in this case, it may be preferable to use a preference on CustomerTokenServiceInterface, instead of CustomerTokenService
all instances of CustomerTokenService usage in PHP classes / API endpoints are initiated through CustomerTokenServiceInterface, so that should be safe

Reply:
never-mind the above comment, I only now saw that we didn't actually override the revokeCustomerAccessToken, in which case extending base class makes sense
(can still do the preference on the interface in di.xml)